airbyte-integrations/connectors/source-github/source_github/schemas/organizations.json

```diff
@@ -118,9 +118,27 @@
     "two_factor_requirement_enabled": {
       "type": ["null", "boolean"]
     },
+    "members_allowed_repository_creation_type": {
+      "type": ["null", "string"]
+    },
+    "members_can_create_public_repositories": {
+      "type": ["null", "integer"]
+    },
+    "members_can_create_private_repositories": {
+      "type": ["null", "integer"]
+    },
+    "members_can_create_internal_repositories": {
+      "type": ["null", "integer"]
+    },
     "members_can_create_pages": {
       "type": ["null", "boolean"]
     },
+    "members_can_fork_private_repositories": {
+      "type": ["null", "integer"]
+    },
+    "web_commit_signoff_required": {
+      "type": ["null", "integer"]
+    },
```

Comment:
I think most of these should be `boolean`, according to the docs [here](https://docs.github.com/en/enterprise-server@3.6/rest/orgs/orgs#get-an-organization) (the names tipped me off)

Reply:
oops my fault, I have fixed!

I tried to understand how this could happen because I am using scripts for schema generation stuff
I have realized when I tweaked the schema for this stream I used buggy  `obfuscate.py` script (later it was fixed)

anyway, thank for catching it - schema regenerated!

Reply:
Sweet, happy to have caught something in the script :)